modules/social_features/social_featured_content/templates/node--featured.html.twig

```diff
@@ -122,6 +123,7 @@
           {% if display_submitted %}
             {% embed "node--teaser__field.html.twig" %}
               {%- block field_icon -%} account_circle {%- endblock -%}
+              {% block svg_title %} <title>{% trans %} Author name {% endtrans %}</title> {% endblock %}
```

Comment:
This SVG should be hidden because there's an sr-only line with the same information right after it.

Reply:
All SVG icons displayed via node--teaser__field.html.twig template have a aria-hidden='true' attribute